File: lc1748.java
import java.util.HashMap;
import java.util.Map;

public class lc1748 {
    //    1748. Sum of Unique Elements
    public static void main(String[] args) {
        int nums[] = {1, 1, 1, 1, 1};
        int ans = 0;
        HashMap<Integer, Integer> hm = new HashMap<>();
        for (int i = 0; i < nums.length; i++) {
            if (hm.containsKey(nums[i])) {
                hm.put(nums[i], hm.get(nums[i]) + 1);
            } else
                hm.put(nums[i], 1);
        }
        for (Map.Entry<Integer, Integer> e : hm.entrySet()) {
            if (e.getValue() == 1) {
                ans = ans + e.getKey();
            }
        }
        System.out.println(ans);
    }
}

File: lc1748.py
from typing import List


class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        a=[]
        for i in nums:
            c=0
            for j in nums:
                if i==j:
                    c+=1
            if c==1:
                a.append(i)
        ans=0
        for i in a:
            ans+=i
        return ans